util/image-upload.js

```diff
@@ -0,0 +1,46 @@
+import { By, until } from "selenium-webdriver";
+import { defaultWait } from "../config.js";
+import path from "path";
+
+export async function uploadImage(driver, element_id, imagePath) {
+  const imageField = await driver.findElement(By.id(element_id));
+  const image = path.resolve(imagePath);
+  await imageField.sendKeys(image);
+  await driver.wait(
+    until.elementLocated(
+      By.css('img[src^="data:image/png;base64,iVBORw0KGg"]')
+    ),
+    defaultWait
+  );
+}
+
+export async function selectNounProjectImage(driver, searchText) {
+  // Open noun-project dialog
+  const nounProject_option = await driver.wait(
+    until.elementLocated(By.id("nounProject_span")),
+    defaultWait
+  );
+  await nounProject_option.click();
+  // search for an image
+  const searchIconField = await driver.wait(
```

Comment:
Oh wow I didn't know this was possible! We could probably also improve code in other places of the project